### PR TITLE
output --dump-default-config to stdout if no path is given

### LINKED ISSUE #1988: `--dump-default-config` does not have a default argument
## Steps to reproduce
`$ rustfmt --dump-default-config`

## Expected behaviour
The path defaults to "rustfmt.toml".
As per #1558. Also see [this comment](https://github.com/rust-lang-nursery/rustfmt/pull/1982#discussion_r139875215).

## Actual behaviour
Error:
```
~/dev/rust/hello_cargo/src (master #)$ rustfmt --dump-default-config
Argument to option 'dump-default-config' missing
...
```
## More info
```
~$ rustfmt --version
0.2.6-nightly ( )
```

### PR DESCRIPTION
**Pull request by japaric**
closes #1988

### CODE REVIEW

**Comment by japaric** [comment]
Alternative implementation (write to rustfmt.toml if no path is given) in #2046.

**Comment by nrc** [comment]
Thank you!

File: src/bin/rustfmt.rs [hunk 1]
```diff
@@ -125,11 +125,14 @@ fn make_opts() -> Options {
         "config-help",
         "show details of rustfmt configuration options",
     );
-    opts.optopt(
+    opts.opt(
         "",
         "dump-default-config",
-        "Dumps the default configuration to a file and exits.",
+        "Dumps the default configuration to a file and exits. PATH defaults to rustfmt.toml if \
```
Comment (tamird) [inline src/bin/rustfmt.rs]: this comment is incorrect?

Comment (tamird) [reply src/bin/rustfmt.rs]: Added a commit to correct this in #2060.

Comment (japaric) [reply src/bin/rustfmt.rs]: Whoops, thanks for noticing and fixing this.

**Pull request merged**